README.md

```diff
@@ -5,28 +5,64 @@
 [![Publish Package](https://github.com/capitalone/rubicon/actions/workflows/publish-package.yml/badge.svg)](https://github.com/capitalone/rubicon/actions/workflows/publish-package.yml)
 [![Publish Docs](https://github.com/capitalone/rubicon/actions/workflows/publish-docs.yml/badge.svg)](https://github.com/capitalone/rubicon/actions/workflows/publish-docs.yml)
 
-`rubicon` is a data science tool for capturing all information related to a model
-during its development. With minimal effort, Rubicon's Python library can integrate
-directly into your Python models:
+## Purpose
+
+Rubicon is a data science tool that captures and stores model training and
+execution information, like parameters and outcomes, in a repeatable and
+searchable way. Rubicon's git integration associates these inputs and outputs
+directly with the model code that produced them to ensure full auditability and
+reproducibility for both developers and stakeholders alike. And the Rubicon
+dashboard makes it easy to explore, filter, visualize, and share recorded work.
+
+---
+
+## Components
+
+Rubicon is composed of three parts:
+
+* A Python library for storing and retrieving model inputs, outputs, and
+  analyses to filesystems that’s powered by
+  [fsspec](https://filesystem-spec.readthedocs.io/en/latest/?badge=latest)
+* A dashboard for exploring, comparing, and visualizing logged data built with
+  [dash](https://dash.plotly.com/)
+* And a process for sharing a selected subset of logged data with collaborators
+  or reviewers that leverages [intake](https://intake.readthedocs.io/en/latest/)
+
+## Workflow
```

Comment:
maybe "Workflow" should be the first subsection of "Use" before the example to make it flow a little better

Reply:
i don't want the example to look intimidating, which it might if it starts with a couple of paragraphs -- but maybe we can switch this later based on more feedback